packages/tpetra/core/test/CrsMatrix/CrsMatrix_UnitTests4.cpp

```diff
@@ -1127,12 +1139,12 @@ inline void tupleToArray(Array<T> &arr, const tuple &tup)
         compare.update(ONE,Y1a,-ONE,Y1b,ZERO);
         compare.norm1(norm());
         if (ST::isOrdinal) {TEST_COMPARE_ARRAYS(exact,norm);}
-        else{TEST_COMPARE_FLOATING_ARRAYS(exact,norm,2.0*testingTol<Mag>());}
+        else{TEST_ABSOLUTE_COMPARE_FLOATING_ARRAYS(exact,norm,absNorm*testingTol<Mag>());}
         
         compare.update(ONE,Y2a,-ONE,Y2b,ZERO);
         compare.norm1(norm());
         if (ST::isOrdinal) {TEST_COMPARE_ARRAYS(exact,norm);}
-        else {TEST_COMPARE_FLOATING_ARRAYS(exact,norm,2.0*testingTol<Mag>());}
+        else {TEST_ABSOLUTE_COMPARE_FLOATING_ARRAYS(exact,norm,absNorm*testingTol<Mag>());}
```

Comment:
Have you had a chance to look at the order of magnitude we can expect for `absNorm*testingTol<Mag>()`? And how stringent it is as compared with the error that we were seeing?

Reply:
@maartenarnst Yes, absNorm ended up being fairly large (roughly 2x10^5 on all the ranks) which explains why the absolute error seemed high (1x10^-11) between two consecutive applies.